Rakefile.d/cucumber.rake

```diff
@@ -17,14 +17,16 @@ namespace :cucumber do
     sh "BUNDLE_GEMFILE=#{current_gemfile} bundle show cucumber > /tmp/cukes_version"
     sh "BUNDLE_GEMFILE=#{current_gemfile} bundle show sqlite3 > /tmp/sqlite_version"
     cukes_version=File.read('/tmp/cukes_version').split('-').last.chomp
-    sqlite_version=File.read('/tmp/sqlite_version').split('-').last.chomp
+    # Example 'sqlite_version' output:
+    #   /Users/username/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/sqlite3-1.6.8-arm64-darwin
+    sqlite_version=File.basename(File.read('/tmp/sqlite_version')).split('-')[1].chomp
```

Comment:
"version" can include architecture information, so `.split('-').last` won't work well for these cases (will return invalid string for version, eg: `darwin`)

Reply:
Could `Gem::Version.new('3.0.1-arm64')` be used here instead?

Reply:
Actually yes, we can get version number a little bit cleaner (without writing to the /tmp file).

Fixed this 🙆‍♂️